indra/statements/statements.py

```diff
@@ -858,117 +858,94 @@ class Dephosphorylation(RemoveModification):
     >>> erk = Agent('MAPK1')
     >>> dephos = Dephosphorylation(dusp6, erk, 'T', '185')
     """
-    pass
 
 
 class Hydroxylation(AddModification):
     """Hydroxylation modification."""
-    pass
 
 
 class Dehydroxylation(RemoveModification):
     """Dehydroxylation modification."""
-    pass
 
 
 class Sumoylation(AddModification):
     """Sumoylation modification."""
-    pass
 
 
 class Desumoylation(RemoveModification):
     """Desumoylation modification."""
-    pass
 
 
 class Acetylation(AddModification):
     """Acetylation modification."""
-    pass
 
 
 class Deacetylation(RemoveModification):
     """Deacetylation modification."""
-    pass
 
 
 class Glycosylation(AddModification):
     """Glycosylation modification."""
-    pass
 
 
 class Deglycosylation(RemoveModification):
     """Deglycosylation modification."""
-    pass
 
 
 class Ribosylation(AddModification):
     """Ribosylation modification."""
-    pass
 
 
 class Deribosylation(RemoveModification):
     """Deribosylation modification."""
-    pass
 
 
 class Ubiquitination(AddModification):
     """Ubiquitination modification."""
-    pass
 
 
 class Deubiquitination(RemoveModification):
     """Deubiquitination modification."""
-    pass
 
 
 class Farnesylation(AddModification):
     """Farnesylation modification."""
-    pass
 
 
 class Defarnesylation(RemoveModification):
     """Defarnesylation modification."""
-    pass
 
 
 class Geranylgeranylation(AddModification):
     """Geranylgeranylation modification."""
-    pass
 
 
 class Degeranylgeranylation(RemoveModification):
     """Degeranylgeranylation modification."""
-    pass
 
 
 class Palmitoylation(AddModification):
     """Palmitoylation modification."""
-    pass
 
 
 class Depalmitoylation(RemoveModification):
     """Depalmitoylation modification."""
-    pass
 
 
 class Myristoylation(AddModification):
     """Myristoylation modification."""
-    pass
 
 
 class Demyristoylation(RemoveModification):
     """Demyristoylation modification."""
-    pass
 
 
 class Methylation(AddModification):
     """Methylation modification."""
-    pass
 
 
 class Demethylation(RemoveModification):
     """Demethylation modification."""
-    pass
```

Comment:
In what way were these passes a problem here?

Reply:
There's incorrect or wrong with using `pass` here, but it's only necessary inside a control structure where there are no other interpretable statements. Since there's a docstring, this class definition isn't empty and it doesn't need a pass. I think that having these short class names really shows the elegant of the Statement hierarchy, and this is a 1/3 improvement on code efficiency! 

I really shouldn't have put this in this PR though... Let me know and I will revert these changes.

Related: my favorite use of pass is:

```python
try:
    do_something_dangerous()
except Exception:
    pass
```

If you think that's funny, check out https://github.com/ajalt/fuckitpy for a good laugh